pipeline/routing/ppr.py

```diff
@@ -32,7 +34,128 @@ def insertPathsElementsRef(cur, edges, path_counter):
             )
 
 
-def insertPGSQL(cur,insertRoutes,start, stop ,path_counter):
+def insertPathsLinks(cur, pathLink, id_from, id_to):
+    edgeList = [f"{edge[0]} {edge[1]}" for edge in pathLink]
+    linestring = "LINESTRING(" + ",".join(edgeList) + ")"
+    
+    if id_from < id_to:
+        smaller_node_id = id_from
+        bigger_node_id = id_to
+    else:
+        smaller_node_id = id_to
+        bigger_node_id = id_from
+        
+    # use INSERT INTO ... ON CONFLICT DO NOTHING to avoid duplicate entries
+    cur.execute(
+        'INSERT INTO path_links (path_id, smaller_node_id, bigger_node_id, geom) VALUES (%s, %s, %s, ST_GeomFromText(%s, 4326)) ON CONFLICT DO NOTHING',
+        (1, smaller_node_id, bigger_node_id, linestring)
+    )
+    
+    # empty pathLink
+    pathLink.clear()
```

Comment:
I feel like a function that is supposed to insert something it gets via parameters is not supposed to clear any of its parameters.